Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/WANDPowderReduction.py

```diff
@@ -200,40 +197,47 @@ def PyExec(self):
                     EnableLogging=False,
                 )
 
-            # conjoin
-            if n < 1:
-                CloneWorkspace(
-                    InputWorkspace=_ws_resampled,
-                    OutputWorkspace="__ws_conjoined",
+            if summing == True:
+                # conjoin
+                if n < 1:
+                    CloneWorkspace(
+                        InputWorkspace=_ws_resampled,
+                        OutputWorkspace="__ws_conjoined",
+                        EnableLogging=False,
+                    )
+                else:
+                    ConjoinWorkspaces(
+                        InputWorkspace1="__ws_conjoined",
+                        InputWorkspace2=_ws_resampled,
+                        CheckOverlapping=False,
+                        EnableLogging=False,
+                    )
+
+        # END_FOR: prcess_spectra
+
+        # Step_3: sum all spectra
+        # ref: https://docs.mantidproject.org/nightly/algorithms/SumSpectra-v1.html
+        if summing == True:
+            if cal is not None:
+                SumSpectra(
+                    InputWorkspace="__ws_conjoined",
+                    OutputWorkspace=outWS,
+                    WeightedSum=True,
+                    MultiplyBySpectra=not bool(cal),
                     EnableLogging=False,
                 )
             else:
-                ConjoinWorkspaces(
-                    InputWorkspace1="__ws_conjoined",
-                    InputWorkspace2=_ws_resampled,
-                    CheckOverlapping=False,
+                SumSpectra(
+                    InputWorkspace="__ws_conjoined",
+                    OutputWorkspace=outWS,
+                    WeightedSum=True,
+                    MultiplyBySpectra=True,
                     EnableLogging=False,
                 )
-        # END_FOR: prcess_spectra
 
-        # Step_3: sum all spectra
```

Comment:
This whole bit (old 219 - new L240) should be something like
```
if summing:
  # old code with SumSpectra
else:
  # new code for GroupWorkspaces
```

Reply:
(mostly) ignore this comment.

Changing `if summing == False:` to `else:` would make it clearer that you are doing the right thing already.